lms/djangoapps/courseware/views.py

```diff
@@ -6,13 +6,14 @@
 import urllib
 import json
 import cgi
-
 from datetime import datetime
 from collections import defaultdict
+from markupsafe import escape
```

Comment:
I am confused by this import re-ordering. Our import ordering guidelines are here: https://github.com/edx/edx-platform/wiki/Python-Guidelines#imports-order